frontend/src/metabase/components/FieldValuesWidget/utils.ts

```diff
@@ -274,11 +247,11 @@ export function getTokenFieldPlaceholder({
     return getSearchableTokenFieldPlaceholder(
       parameter,
       fields,
-      firstField,
+      fields[0],
```

Comment:
I don't think there's anything wrong with reusing the same variable `firstField`. That emphasizes both line 250 and 254 are using the same variable.

I kind of think this is a step backwards

Reply:
Fair enough, I restored the old code